src/modules/core/components/Button/ThreeDotsButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { MessageDescriptor } from 'react-intl';
+
+import classnames from 'classnames';
+import Button, { Props as DefaultButtonProps } from './Button';
```

Comment:
These should be separated from the `~core` import by an empty line (should be after core inputs together with `import styles from './ThreeDotsButton.css';` import

Reply:
Is this what you meant?
<img width="904" alt="Screen Shot 2022-07-29 at 2 26 17 AM" src="https://user-images.githubusercontent.com/71563622/181658523-90b7a37f-cf49-494a-aca3-736f1e7e9a67.png">
Thanks!